### PR TITLE
Datetime

### PR DESCRIPTION
**Pull request by DannyHatcher-Git**
This is my first Pull request so sorry if I do something wrong.

### CODE REVIEW

File: docs/docs/features/Formulas.md [hunk 1]
```diff
@@ -22,3 +22,33 @@ The root object `db` has the following functions:
 
 - `js` : execute a javascript function that you previously defined in the `js` folder of your table in the database or plugin global settings. (I.E.: `db.js.myFunction( arg1, arg2)`)
 
+## Examples
```
Comment (RafaelGB) [inline docs/docs/features/Formulas.md:25]: I like the idea to include examples

When can include sections. Your PR could be tittled as Datetime and includes the link to Luxon documentation

[luxon](https://moment.github.io/luxon)

**Comment by RafaelGB** [comment]
Thanks for your collaboration!

The PR its cool =)
We can discuss details with comments inside the file itself but I will leave it approved

**Review by RafaelGB — APPROVED** [review]

**Comment by DannyHatcher-Git** [comment]
I am trying to work out how to re-edit the changes I made to add the Luxon doc link.
Working out if I need to do it in my version and do a different pull request, or if I can edit it here somehow.

**Comment by RafaelGB** [comment]
You can just update the same PR

Edit it in your fork and should be updated

**Comment by RafaelGB** [comment]
If you consider that it is ready, we can inaugurate your first PR and upload it to the repository.

**Comment by DannyHatcher-Git** [comment]
I think it is ready now.
I know if I could see that when I first started it would have helped me a lot.

**Comment by RafaelGB** [comment]
Merged!

**Pull request merged**